examples/platforms/cc2650/arm-none-eabi.cmake

```diff
@@ -0,0 +1,52 @@
+#
```

Comment:
(Can be resolved afterwards)
Duplicate code in ${PLATFORM_NAME}/arm-none-eabi.cmake